adapter/prometheus/config/prometheus.proto

```diff
@@ -0,0 +1,22 @@
+// Copyright 2017 Google Inc.
```

Comment:
For the other adapters, we've called this file just config.proto. We should be consistent throughout IMO.